src/main/java/com/refinedmods/refinedstorage/screen/grid/GridScreen.java

```diff
@@ -86,7 +89,6 @@ public void onPageChanged(int page) {
     @Override
     protected void onPreInit() {
         super.onPreInit();
-        this.doSort = true;
```

Comment:
Why did you move this?

Reply:
Oh wow. Forgot this was in here. This was something I forgot to commit in the shiftpreventssorting PR. Totally unnecessary